### PR TITLE
Add operator based on helm charts

### PR DESCRIPTION
**Pull request by lgarciaaco**

### CODE REVIEW

File: operator/deploy/cr.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+apiVersion: apicur.io/v1alpha1
+kind: Apicurito
+metadata:
+  name: apicurito
+spec:
+  replicaCount: 2
```
Comment (heiko-braun) [inline operator/deploy/cr.yaml]: Does need two?

Comment (EricWittmann) [reply operator/deploy/cr.yaml]: No two aren't needed, but it wouldn't hurt.

Apicurito is 100% client only - the server does nothing except serve static html/css/js files.  The great irony is that it scales horizontally **extremely** well....but doesn't need to. :)

**Comment by EricWittmann** [comment]
I don't have any expertise in this area....but it looks ok to me...question mark.

File: operator/deploy/cr.yaml (hunk 1 above)
Comment (lgarciaaco) [reply operator/deploy/cr.yaml]: Well since it is very light I though 2 can enhance high availability at a very low cost, that is why ;)

**Comment by heiko-braun** [comment]
Fair enough ;)

File: operator/deploy/cr.yaml (hunk 1 above)
Comment (paoloantinori) [reply operator/deploy/cr.yaml]: I don't think it's needed at this stage as a default, but I think that this is a nice addition to documentation to show users how they can scale.

**Comment by amysueg** [comment]
Curious whether or not this item should also be considered part of the Fuse/Apicurio operator epic,https://github.com/syndesisio/syndesis/issues/4154 ?
@heiko-braun

**Comment by heiko-braun** [comment]
@lgarciaaco Please rebase, test and make sure this get's merged.

**Comment by lgarciaaco** [comment]
@EricWittmann would you mind to merge? I don't seem to have the rights

**Comment by EricWittmann** [comment]
@lgarciaaco The only reason I didn't merge this long ago is that I was hoping for a review/approval from someone who knows more about operators than I do. :)

**Comment by tplevko** [comment]
@EricWittmann regarding the apicurito operator, is the operator image going to be available on dockerhub together with the other two apicurito images?

**Comment by EricWittmann** [comment]
@tplevko To answer that I need to know what an operator image is.... :)

**Comment by tplevko** [comment]
@EricWittmann sorry, maybe a better question for @lgarciaaco. This could be added to the apicurio build pipeline.

**Pull request merged**